CHANGELOG.md

```diff
@@ -5,6 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## 2.2.6 (2021-05-10)
```

Comment:
thanks for adding this! , i think will be better to release one version with all this, `v2.3.0` bumping the minor as the other integrations